### PR TITLE
feat: Migration - Code patterns ALA-1104

### PR DESCRIPTION
**Pull request by nicklem**
### :eyes: Live preview
https://ala-1104-doc-document-migrated-page--docs-codacy.netlify.app/repositories-configure/configuring-code-patterns/

### :construction: To do
- [x] Review all TODOs
- [x] Audit screenshots
- [x] Audit docs for references to page

### CODE REVIEW

File: docs/repositories-configure/configuring-code-patterns.md [hunk 1]
```diff
@@ -54,26 +57,26 @@ To configure the tools and code patterns for a repository using the Codacy UI:
...
 To use a configuration file for a static analysis tool:
 
 1.  Push the configuration file to the root of the [default Codacy branch](managing-branches.md).
 
-1.  Open the repository **Code patterns** page, select the tool of interest, and select the option **Configuration file**.
+1.  Open the repository **Code patterns** page, select the tool of interest, and activate the toggle to use a configuration file.
 
     !!! note
-        -   Codacy uses the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results take those changes into account.
+        -   Codacy uses the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results take those changes into account.<!-- TODO ALA-1104 confirm this is still the case -->
```
Comment (nicklem) [inline docs/repositories-configure/configuring-code-patterns.md:71]: @codacy/alabama Is this still the case?

**Review by nicklem — COMMENTED** [review]
Some comments for review

File: docs/repositories-configure/configuring-code-patterns.md [hunk 2]
```diff
@@ -26,19 +23,25 @@ To configure the tools and code patterns for a repository using the Codacy UI:
...
 
-    To see an explanation of the issues that a pattern detects and how to fix them, click **Show details**. Some patterns also allow you to configure the rules for detecting issues.
+    To see an explanation of the issues that a pattern detects and how to fix them, click the respective dropdown arrow.
 
     !!! tip
-        -   To enable a group of code patterns, use the filter to select the relevant group of patterns and click **Enable all**. For example, to enable all Security patterns, click the **Security** filter and then click **Enable all**.
+        -   To enable a group of code patterns, use the filter to select the relevant group of patterns and click the checkbox in the header of the patterns list.
 
-        -   Codacy displays the tag **New** for one month next to the name of newly added code patterns.
+        -   Codacy displays the tag **New** for one month next to the name of newly added code patterns.<!-- TODO ALA-1104 confirm this is still valid -->
```
[the remark this answers is not in the document]
Comment (alerizzo) [reply docs/repositories-configure/configuring-code-patterns.md:33]: I don't think so.... And if so, idk if it's relevant for the docs.

File: docs/repositories-configure/configuring-code-patterns.md [hunk 3]
```diff
@@ -54,26 +57,26 @@ To configure the tools and code patterns for a repository using the Codacy UI:
 !!! note
     -   After activating a configuration file for a tool, Codacy uses that configuration file even if you [exclude it from Codacy analysis](ignoring-files.md).
 
-    -   When [using a tool configuration file alongside a coding standard](../organizations/using-coding-standards.md#using-with-tool-configuration), the configuration file controls the code patterns, while the coding standard controls whether the tool is enabled or disabled.
+    -   When [using a tool configuration file alongside a coding standard](../organizations/using-coding-standards.md#using-with-tool-configuration), the configuration file controls the code patterns, while the coding standard controls whether the tool is enabled or disabled.<!-- TODO ALA-1104 confirm this is still the case -->
```
[the remark this answers is not in the document]
Comment (alerizzo) [reply docs/repositories-configure/configuring-code-patterns.md:60]: It is still the case

File: docs/repositories-configure/configuring-code-patterns.md [hunk 4]
```diff
@@ -54,26 +57,26 @@ To configure the tools and code patterns for a repository using the Codacy UI:
...
 
     !!! note
-        -   Codacy uses the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results take those changes into account.
+        -   Codacy uses the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results take those changes into account.<!-- TODO ALA-1104 confirm this is still the case -->
 
-        -   If Codacy analyzes a branch that doesn't include the configuration file, Codacy reverts to using the code patterns configured for the tool before you selected the option **Configuration file** on the Code patterns page.
+        -   If Codacy analyzes a branch that doesn't include the configuration file, Codacy reverts to using the code patterns configured for the tool before you selected the option **Configuration file** on the Code patterns page.<!-- TODO ALA-1104 confirm this is still the case -->
```
[the remark this answers is not in the document]
Comment (alerizzo) [reply docs/repositories-configure/configuring-code-patterns.md:73]: Yes

File: docs/repositories-configure/configuring-code-patterns.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/repositories-configure/configuring-code-patterns.md:33]:
```suggestion
        -   Codacy displays the tag **New** for one month next to the name of newly added code patterns.
```

File: docs/repositories-configure/configuring-code-patterns.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/repositories-configure/configuring-code-patterns.md:60]:
```suggestion
    -   When [using a tool configuration file alongside a coding standard](../organizations/using-coding-standards.md#using-with-tool-configuration), the configuration file controls the code patterns, while the coding standard controls whether the tool is enabled or disabled.
```

File: docs/repositories-configure/configuring-code-patterns.md (hunk 1 above)
Comment (nicklem) [reply docs/repositories-configure/configuring-code-patterns.md:71]:
```suggestion
        -   Codacy uses the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results take those changes into account.
```

File: docs/repositories-configure/configuring-code-patterns.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/repositories-configure/configuring-code-patterns.md:73]:
```suggestion
        -   If Codacy analyzes a branch that doesn't include the configuration file, Codacy reverts to using the code patterns configured for the tool before you selected the option **Configuration file** on the Code patterns page.
```

File: docs/repositories-configure/configuring-code-patterns.md [hunk 5]
```diff
@@ -73,7 +76,7 @@ To use a configuration file for a static analysis tool:
 
     ![Using a configuration file](images/code-patterns-config-file.png)
 
-The table below lists the configuration file names that Codacy detects and supports for each tool:
+The table below lists the configuration file names that Codacy detects and supports for each tool:<!-- TODO ALA-1104 confirm this is still the case -->
```
Comment (nicklem) [inline docs/repositories-configure/configuring-code-patterns.md:79]:
```suggestion
The table below lists the configuration file names that Codacy detects and supports for each tool:
```

**Review by alerizzo — APPROVED** [review]

**Pull request merged**